lighthouse-extension/fs-transform.js

```diff
@@ -1,37 +0,0 @@
-/**
```

Comment:
had no effect on the output. Maybe it was required for some ancient version of speedline that pulled in catapult?